packages/date-picker/src/panel/date.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <transition name="el-zoom-in-top" @after-enter="handleEnter" @after-leave="handleLeave">
+    <!-- Prevent mousedown from stealing focus from date input -->
     <div
+      @mousedown.prevent="noop"
```

Comment:
Do you think it would be more clear to do something like `@mousedown="handleContainerMousedown"` and in that method do a `.preventDefault()` call and put the comment there? I think that would tie the prevent and comment together a little better. Also would eliminate the need for a pesky `noop` method.

Reply:
Yeah, that is better. Why "Container" though?